src/main/scala/org/broadinstitute/dsde/firecloud/service/DataUseRestrictionSupport.scala

```diff
@@ -12,6 +12,30 @@ import spray.json._
 
 import scala.util.Try
 
+object ConsentCodes extends Enumeration {
+  val GRU = "GRU"
+  val HMB = "HMB"
+  val NCU = "NCU"
+  val NPU = "NPU"
+  val NMDS = "NMDS"
+  val NAGR = "NAGR"
+  val NCTRL = "NCTRL"
+  val RSPD = "RS-PD"
+  val IRB = "IRB"
+  val RSG = "RS-G"
+  val RSFM = "RS-FM"
+  val RSM = "RS-M"
+  val DSURL = "DS_URL"
+  val RSPOP = "RS-POP"
+  val DS = "DS"
+
+  val booleanCodes = Seq(GRU, HMB, NCU, NPU, NMDS, NAGR, NCTRL, RSPD, IRB)
+  val genderCodes = Seq(RSG, RSFM, RSM)
+  val duRestrictionFieldNames = booleanCodes ++ genderCodes ++ Seq(DSURL, RSPOP)
+  val allDurFieldNames = duRestrictionFieldNames ++ Seq(DS)
+  val diseaseLabelsAttributeName: AttributeName = AttributeName.withLibraryNS(DS)
+}
```

Comment:
Shoved everything into an enum. If we're good with this, I can replace other stuff in this file with the vals in here.

Reply:
It would be nice to somehow reuse this enum, and/or colocate it, with the similar fields in `DataUse.ResearchPurpose`. The `DataUse` object was intended to be the home for exactly this kind of information.